examples/sample_manifest_extended.json

```diff
@@ -0,0 +1,169 @@
+{
+"metadata": {
+    "@type": "http://bib.schema.org/Audiobook",
+    "identifier": "https://librivox.org/flatland-a-romance-of-many-dimensions-by-edwin-abbott-abbott/",
+    "title": "Flatland: A Romance of Many Dimensions",
+    "author": [{
+        "name": {
+            "ru": "Эдвин Эбботт Эбботт",
+            "en": "Edwin Abbott Abbott"
+        },        
+        "sort_as": {
+            "ru": "Эбботт, Эдвин Эбботт",
+            "en": "Abbott, Edwin Abbott"
+        },
+    }, {
+        "name": "Bob Braggert",
+        "sort_as": "Braggert, Bob"
+    }],
+    "narrator": [
+        {
+        "name": "Abe Narrator",
+        },
+        {
+        "name": "Beatrice Narrator",
+        }
+    ],
+    "illustrator": [
+        {
+        "name": "Abe Illustrator",
+        }
+    ],
+    "translator": [
+        {
+        "name": "Abe Illustrator",
+        }
+    ],
+    
+    "language": "en-US",
+    "description": "Flatland: A Romance of Many Dimensions is an 1884 science fiction novella by the English schoolmaster Edwin Abbott Abbott. As a satire, Flatland offered pointed observations on the social hierarchy of Victorian culture. However, the novella's more enduring contribution is its examination of dimensions; in a foreword to one of the many publications of the novella, noted science writer Isaac Asimov described Flatland as \"The best introduction one can find into the manner of perceiving dimensions.\" As such, the novella is still popular amongst mathematics, physics and computer science students. (Summary by Wikipedia)",
+    "subject": ["General Fiction", "Satire", "Science Fiction"],
+    
+    "schema:typicalAgeRange": "7-11",
+    "publisher": "Librivox",
+    "published": "2008-10-12",
+    
+    "modified": "2016-08-11T19:32:18Z",
+    
+    "belongs_to": {
+      "collection": "Math for All",
+      "series": {
+        "name": "Geometry Trilogy",
+        "position": 4
+      }
+    },
+    
+    "duration": 15153,
+    "type": "audio/mpeg",
+    "bitrate": 128,
+    "samplerate": "8.5",
+    "channels": 2,
+    
+    "schema:hasPart": [
+        {
+            "@type": "http://schema.org/VisualArtwork",
+            "schema:url": "cover.jpg",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        },
+        {
+            "@type": "http://schema.org/Book",
+            "schema:url": "this_is_for_copyright_on_text_copy",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        },
+        {
+            "@type": "http://bib.schema.org/Audiobook",
+            "schema:url": "this_is_for_copyright_on_performance",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        }
+    ],
+    
+    "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/",
+    
+    "schema:hasPart": [
```

Comment:
Ah, that is an artifact of me having to edit the file externally and then copy-pasting into here.  Removed.